frontend/src/components/editor/notebook-cell.tsx

```diff
@@ -580,6 +580,7 @@ const EditableCellComponent = ({
             ref={cellContainerRef}
             {...cellDomProps(cellId, cellData.name)}
           >
+            <CellLeftSideActions cellId={cellId} actions={actions} />
```

Comment:
@Light2Dark could this go under `tray`? does it break anything? currently putting it here messes with the css rounding of the editor